examples/requirements_develop.py

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env python3
+
+# Installs wheels for current develop version 
+import sys, os, subprocess
+parent_dir = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+os.chdir(parent_dir)
+sys.path.insert(1, parent_dir)
+import find_version
+
+# Get package version
+version = find_version.get_package_version()
+git_commit = subprocess.check_output(['git', 'rev-parse', 'HEAD']).decode('UTF-8').strip()
+final_version = version + "+" + git_commit
```

Comment:
I'd say remove version info from development builds, so one can easily install in the future any wheel from any commit version (after we make the change) without knowing the version info, and not having to use the script from the specific branch. 
This also would make depthai lib easier to upgrade with just -U, not having to have --force_reinstall to get latest release.